tutor4points/core/templates/requests.html

```diff
@@ -13,52 +13,142 @@
 
 {% block content %}
 
-{% for request in requests_received %}
-
-<div class = "request-container" id = "request-{{request.id}}" data-requestid="{{request.id}}">
-    {{request.tutee.id}}
-    {{request.date_sent_request}} <br>
-    {{request.tutor_date}}  <br>
-    {{request.tutor_time}}  <br>
-    {{request.class_name}}  <br>
-    {{request.location}}  <br>
-    {{request.tutee_comment}} <br> <br>
-    <div>
-        <button type ="submit" name="btn-accept-request" class="btn-secondary" data-toggle= "modal" data-target="#modal-accept-request">Accept</button>
-        <button type ="submit" name="btn-reject-request" class="btn-secondary" data-toggle= "modal" data-target="#modal-decline-request">Decline</button>
+<div class="scroll-box">
+    <h2>Inbox</h2>
+    {% if requests_received %}
+    <div class="requests-items">
+        {% for request in requests_received %}
+
+        <div class="request-container d-flex" id="request-{{request.id}}" data-requestid="{{request.id}}"
```

Comment:
I noticed that there seems to be more margin around the items in the _Sent_ box than the _Inbox_. It'll be great if they have the same margins.

Reply:
Added the margin in main